reference/provided-images.md

```diff
@@ -25,9 +25,23 @@ The following table lists supported images available on the official image serve
 | `jammy:android13:arm64`     | AOSP     | 13              | 22.04          | 1.16.0 |
 | `jammy:android12:amd64`     | AOSP     | 12              | 22.04          | 1.14.0 |
 | `jammy:android12:arm64`     | AOSP     | 12              | 22.04          | 1.14.0 |
+| `noble:aaos15:amd64`        | AAOS     | 15              | 24.04          | 1.26.0 |
```

Comment:
We don't enable the aosp and aaos running on noble based lxd images, do we? @ajanon @morphis